### ISSUE TITLE
Incorrect newlines added to CSS template literal

### ISSUE THREAD

**Issue by wgoodall01** [prettier/prettier#5886]
**Prettier 1.16.4**
[Playground link](https://prettier.io/playground/#N4Igxg9gdgLgprEAuc0DOMAEAbaBzAfUigBNMBeTGAJwFc4BuAHShYB4MBPbOTAKzQAPAHzAABiwB0YbAEM0aYCyYwAZj0EBaEgEtqcMDB3QkmSNloBbKABJguKIWJkAZC5z4i0V+4dPvmAD8mEwgmvoAbnDUaHChmKahoQC+zFDJLGLJbAD0XDzCIAA0IBAADkboyKCy1NQQAO4ACrUIaMggshEQOiTFIABG1LJgANZwMADKZSM6jsg09CUAFjCW2ADqyzrwaDNgcJNtOzoRO5wdYAr9c7HUME3DeJayyKqy2LElAoIAQsNjCaTWSWOAAGTmcDeHy+IB+kzmeB4AEVaBB4NDPnASjMYtEOgNZANuNB+mVqHMYBtejBlsgABwABhx9ViG2GZQ65Lgdyi-X0AEdaHo4I9ZM9Xkh3liSrFLDoFnRsSA0IiUWiMVKYcqYETqSRacgAEwlGiyHTYREAYQglheHSg0ChJVosQAKkT2lqsclkkA)
```sh
--parser babylon
```

When using JS template literals inside the multiparser, line breaks before long template expressions can add invalid spaces/newlines to CSS. 

Prettier sometimes adds a line break before the opening `${`, which can change e.g. `flex-direction: row-reverse;` to something like `flex-direction: row -reverse`.

This breaks the syntax of the rendered CSS.

**Input:**
```jsx
const long_cond = true;

<style jsx>{`
.class{
	flex-direction: column${long_cond && long_cond && long_cond ? "-reverse" : ""};
}
`}</style>
```

**Output:**
```jsx
const long_cond = true;

<style jsx>{`
  .class {
    flex-direction: column
      ${long_cond && long_cond && long_cond ? "-reverse" : ""};
  }
`}</style>;

```

With the template substitution, this eventually gets inserted into the DOM as the following css:

```css
  .class {
    flex-direction: column -reverse
  }
```

Note the space between `column` and `-reverse`.

**Expected behavior:**
```jsx
const long_cond = true;

<style jsx>{`
  .class {
    flex-direction: column${
        long_cond && long_cond && long_cond 
        ? "-reverse" 
        : ""};
  }
`}</style>;
```

...which should produce the following css:

```css
  .class {
    flex-direction: column-reverse
  }
```

**Comment by lydell** [comment]
Yeah, interpolations in CSS are super hard because unlike good ol’ Sass there are no rules on where they are allowed.

A workaround is to not try to be so super DRY (which IMO is more readable).

**Prettier 1.16.4**
[Playground link](https://prettier.io/playground/#N4Igxg9gdgLgprEAuc0DOMAEAbaBzAfUigBNMBeTGAJwFc4BuAHShYB4MBPbOTAKzQAPAHzAABiwB0YbAEM0aYCyYwAZj0EBaEgEtqcMDB3QkmACTBcUQsTIAyOznxFo9x1ZuvMAfkxNU2LQAtlCa+gBucNRocP6Ypv6QgSH+AL7MUKksYqlsAPRcPMIgADQgEAAORujIoLLU1BAA7gAK9QhoyCCy4RA6JKUgAEbUsmAA1nAwAMoVYzrWyDT0ZQAWMEHYAOqrOvBoc2Bw0x17OuF7nF1gCoMLMdQwLaN4QbLIqrLYMWUCggBCowmU2msiCcAAMgs4B8vj8QH9pgs8DwAIq0CDwWHfOBlObRKJdIayIbcaCDCrUBYwLb9GCrZAADgADHjGjEtqMKl1KXAHpFBvoAI60PRwZ6yV7vJCfHFlGJBHRLOi4kBoZFojFYmVw1UwEm0kj05AAJjKNFkOmwyIAwhAgm8ulBoDCyrQYgAVEmdHU41KpIA)
```sh
--parser babylon
```

**Input:**
```jsx
const long_cond = true;

<style jsx>{`
.class{
	flex-direction: ${long_cond && long_cond && long_cond ? "column-reverse" : "column"};
}
`}</style>
```

**Output:**
```jsx
const long_cond = true;

<style jsx>{`
  .class {
    flex-direction: ${long_cond && long_cond && long_cond
      ? "column-reverse"
      : "column"};
  }
`}</style>;

```

**Comment by wgoodall01** [comment]
This is the solution I ended up implementing. However, I only found this
after an automatic format of a project caused a regression in the layout,
so it's still probably worth fixing for peace-of-mind with large changes.

**Comment by evilebottnawi** [comment]
@lydell sometimes i think we should output values as is without trying account nodes
